CONTRIBUTING.md

```diff
@@ -22,6 +22,14 @@ Set up AWS credentials: environment variables (export AWS_ACCESS_KEY_ID and
 AWS_SECRET_ACCESS_KEY), populate the ~/.aws/credentials file, or use an
 IAM instance profile on an EC2 instance.
 
+Rusoto codegen depends on botocore.  Update the git submodule via:
```

Comment:
As mentioned in https://github.com/rusoto/rusoto/issues/518 we may want to split these instructions off.